user/pages/03.commerce2/02.developer-guide/02.install-update/01.requirements/docs.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Requirements
+taxonomy:
+    category: docs
+---
+
+ ### Requirements
+
+ -Commerce 2.x requires Drupal 8.5.0 or newer.
+
+ Unless you are building a very simple site, you will normally want to develop
```

Comment:
We should recommend they always develop locally, to properly take advantage of Drupal's configuration management. Or just remove this in general